microsoft-edge/progressive-web-apps-chromium/pwabuilder.md

```diff
@@ -0,0 +1,27 @@
+---
+description: PWABuilder.
+title: PWABuilder
+author: MSEdgeTeam
+ms.author: msedgedevrel
+ms.date: 09/16/2021
+ms.topic: conceptual
+ms.prod: microsoft-edge
+keywords: progressive web apps, PWA, Edge, Windows, PWABuilder, web manifest, service worker, push
+---
+# PWABuilder
```

Comment:
Can you tell me more about the rationale behind adding a new page for PWABuilder here? 
We already have a section about using PWABuilder in other places, and this short page doesn't seem to be adding a lot.